src/components/tasks/card/index.tsx

```diff
@@ -258,13 +262,171 @@ const Card: FC<Props> = ({
         actions.onEditRoute();
     };
 
-    return (
-        <div
-            className={`
+    // show redesign only on dev
+    if (dev)
+        return (
+            <div
+                className={`
         ${classNames.card}
+        ${classNames.card_updated}
         ${isLoading && classNames.pointerEventsNone}
         ${isTaskOverdue() && classNames.overdueTask}
     `}
+                data-testid="task-card"
+            >
+                {/* loading spinner */}
+                {isLoading && (
+                    <div className={classNames.loadingBg}>
+                        <div className={classNames.spinner}>
+                            <span className={classNames.screenReaderOnly}>
+                                loading
+                            </span>
+                        </div>
+                    </div>
+                )}
+
+                <div className={classNames.cardItems}>
+                    <h2
+                        className={classNames.cardTitle}
+                        contentEditable={shouldEdit}
+                        onKeyPress={(e) => handleChange(e, 'title')}
+                        role="button"
+                        tabIndex={0}
+                    >
+                        {cardDetails.title}
+                    </h2>
+                    {/* progress bar */}
+                    <div className={classNames.progressContainerUpdated}>
```

Comment:
Similarly, progress bar could be a separate component itself

Reply:
done